index.html

```diff
@@ -63,120 +68,131 @@ <h3 class="connect">LET'S CONNECT</h3>
       <section class="works">
 
         <!-- card 1 -->
-        <div class="card card1">
+        <div class="card card1 odd">
           <img class="card-cover" src="src/images/card-cover1.png" alt="">
-          <h2 class="card-title">Tonic</h2>
-          <ul class="card-details">
-            <li class="prim-detail-cap">CANOPY</li>
-            <li><img src="src/icons/counter.svg" alt=""></li>
-            <Li class="sec-detail-cap">Back End Dev</Li>
-            <li><img src="src/icons/counter.svg" alt=""></li>
-            <li class="sec-detail-cap">2015</li>
-          </ul>
-          <p class="description">
-            A daily selection of privately personalized reads; no accounts or sign-ups required.
-          </p>
-          <ul class="techstack">
-            <li>html</li>
-            <li>css</li>
-            <li>javascript</li>
-          </ul>
-          <button>See Project</button>
+          <div class="card-info">
+            <h2 class="card-title">Tonic</h2>
+            <ul class="card-details">
+              <li class="prim-detail-cap">CANOPY</li>
+              <li><img src="src/icons/counter.svg" alt=""></li>
+              <Li class="sec-detail-cap">Back End Dev</Li>
+              <li><img src="src/icons/counter.svg" alt=""></li>
+              <li class="sec-detail-cap">2015</li>
+            </ul>
+            <p class="description">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="techstack">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button>See Project</button>
```

Comment:
[OPTIONAL] Please add `cursor: pointer` to all the buttons because the cursor is a pointer that indicates a link. Typically an image of a pointing hand. The program is busy in the background, but the user can still interact with the interface (in contrast to waiting). :slightly_smiling_face: